FakeItEasy.Analyzers.sln

```diff
@@ -8,7 +8,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		.gitattributes = .gitattributes
 		.gitignore = .gitignore
 		.mailmap = .mailmap
-		appveyor.yml = appveyor.yml

```

Comment:
We did not add the `ci.yml`?

Reply:
because I forgot! Also didn't do it in FakeItEasy...

Reply:
I'm forever forgetting to add files to the sln. More trouble than it's worth, I sometimes think.